data/anaconda.conf

```diff
@@ -56,6 +56,9 @@ system_root = /mnt/sysroot
 #
 default_on_boot = NONE
 
+# Enable installation of the network configuration on the target system.
+enable_installation = True
```

Comment:
I'm not sure if the name is enough to understand what it does. Not a blocker though.

Reply:
Maybe it could be in the `Installation Target` section as `can_change_network` or `can_configure_network`. It would nicely complement the `Installation System` section that has similar "rules" for the installation environment.

Reply:
The latter sounds good. But your call :)

Reply:
Fixed.